ios-apps-master/Beer_Diary/Beer_Diary/Add_Beer.swift

```diff
@@ -0,0 +1,256 @@
+//
+//  Add_Beer.swift
+//  Beer_Diary
+//
+//  Created by Piotr Mielcarzewicz on 13/11/16.
+//  Copyright © 2016 Piotr Mielcarzewicz. All rights reserved.
+//
+
+import UIKit
+
+struct BeerItem
+{
+    var name = ""
+    var brewery = ""
```

Comment:
Don't set empty string as the default everywhere, because you lose the information that compiler gives you that you forgot to set something, e.g. you copy the object and you forget to set the `country`, it would be an error, but now it is ignored and `country` is ""